src/pt/blackoutcomics/src/eu/kanade/tachiyomi/extension/pt/blackoutcomics/BlackoutComics.kt

```diff
@@ -97,8 +97,8 @@ class BlackoutComics : ParsedHttpSource() {
     // =========================== Manga Details ============================
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
         val row = document.selectFirst("section > div.container > div.row")!!
-        thumbnail_url = row.selectFirst("img:not(.hidden)")?.absUrl("src")
-        title = row.selectFirst("div.trailer-content > h2:not(.hidden)")!!.text()
+        thumbnail_url = row.selectFirst("img.img-recen-add2")?.absUrl("src")
+        title = row.selectFirst("div.trailer-content > h2:not(.tachiyomikappa)")!!.text()
```

Comment:
Make sure you're using the same method to hide elements as on the site. Can always find the class name from css/js if they keep changing it.

Reply:
The source adds an extra element (hidden element) at the same level as the resource you are looking for.

The previous solution reversed the selection logic, it does not contain the hiding class, so it is the resource(thumbnail, title) to be used, because the resource in some cases does not have a css selector

Reply:
I don't think there's a comprehensive enough solution.
It's like a Tom & Jerry.